src/main/pages/che-7/administration-guide/proc_collecting-che-metrics-with-prometheus.adoc

```diff
@@ -0,0 +1,49 @@
+[id="collecting-che-metrics-with-prometheus_{context}"]
+= Collecting Che metrics with Prometheus
+
+Prometheus is a monitoring system that collects metrics in real time and stores them in a time series database.
+
+Prometheus comes with a console accessible at the `9090` port of the application pod. By default, a template provides an existing *service* and a *route* to access it. It can be used to query and view metrics.
+
+image::monitoring/monitoring-che-prometheus-console.png[link="{imagesdir}/monitoring/monitoring-che-prometheus-console.png"]
+
+== Prometheus terminology
+
+Prometheus offers:
+
+counter:: the simplest numerical type of metric whose value can be only increased. A typical example is counting the amount of HTTP requests that go through the system.
+
+gauge:: numerical value that can be increased or decreased. Best suited for representing values of objects.
+
+histogram:: TODO
```

Comment:
histogram:: a more complex metric that is suited for performing observations. Metrics are collected and grouped in configurable buckets, which allwos to present the results, for instance, in a form of a heatmap.